Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMObjectsMappingWizardPage.xaml.cs

```diff
@@ -106,8 +106,7 @@ private void StartTimer()
             elapsedTime = TimeSpan.Zero;
 
             // Start the timer
-            timer.Start();
-            mWizard.mPomLearnUtils.StartLearningTime();
+            timer.Start();        
```

Comment:
Add Try catch to handle unexpected errors

Reply:
done